README.md

```diff
@@ -176,7 +176,10 @@ sources: "BBC,NPR,FOX,CNBC,CNN"
 The way summarization works can be configured as follows:
 
 ```yaml
-# Default language for summary response (if not specified defaults to "auto)
+# Hide or show summary (if not specified defaults to True)
```

Comment:
Perhaps clarify in the doc that it not only hides or shows the summary, but if false this also prevents the call to the summarization API.